src/preprocessing/passes/symmetry_breaker.h

```diff
@@ -0,0 +1,67 @@
+/*********************                                                        */
+/*! \file symmetry_breaker.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **  Paul Meng, Andrew Reynolds
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2018 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Symmetry breaker for theories
+ **/
+
+#ifndef __CVC4__PREPROCESSING_PASSES_SYMMETRY_BREAKER_H_
+#define __CVC4__PREPROCESSING_PASSES_SYMMETRY_BREAKER_H_
+
+#include <map>
+#include <string>
+#include <vector>
+#include "expr/node.h"
+
+namespace CVC4 {
+namespace preprocessing {
+namespace passes {
+
+class SymmetryBreaker
+{
+ public:
+  /**
+   * Constructor
+   * */
+  SymmetryBreaker()
+  {
+    d_trueNode = NodeManager::currentNM()->mkConst<bool>(true);
+    d_falseNode = NodeManager::currentNM()->mkConst<bool>(false);
+  }
+
+  /**
+   * Destructor
+   * */
+  ~SymmetryBreaker() {}
+
+  /**
+   * This is to generate symmetry breaking constraints for partitions in parts.
```

Comment:
Document what parts is (with regards to its properties of specifying a symmetry).